src/Shouldly/Shouldly.csproj

```diff
@@ -34,7 +33,7 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
-    <PackageReference Include="Microsoft.TargetingPack.NETFramework.v4.5.1" Version="1.0.1" ExcludeAssets="All" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net451" Version="1.0.0-preview.1" PrivateAssets="All" />
```

Comment:
You can and probably should omit the `.net451` as it's intended to be referenced by the meta-package, the way NuGet works with meta-packages means that only `net451` will be resolved and downloaded.